src/test/java/com/aws/greengrass/clientdevices/auth/helpers/TestHelpers.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.clientdevices.auth.helpers;
+
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Supplier;
+
+/**
+ * Helpers for generic assertion tooling
+ */
+public final class TestHelpers {
+    public static final long DEFAULT_GENERIC_POLLING_TIMEOUT_MILLIS =
+            TimeUnit.SECONDS.toMillis(10);
+
+    public static final long DEFAULT_POLLING_INTERVAL_MILLIS = 500;
```

Comment:
Are these defaults high enough? What if you try to build this on a raspberry pi?